include/utility/dbus_utility.hpp

```diff
@@ -106,8 +106,8 @@ inline types::PropertyMap getPropertyMap(const std::string& i_service,
 /**
  * @brief An API to read property from Dbus.
  *
- * The caller of the API needs to validate the validatity and correctness of the
- * type and value of data returned. The API will just fetch and retun the data
+ * The caller of the API needs to validate the validity and correctness of the
```

Comment:
why spell correction change got included in this PR? please revert it. Be specific to vpd-tool read implementation only.

Reply:
yes, instead of modifying in utils.hpp, I did in wrong file.
I removed this file changes.